src/components/global/Column.jsx

```diff
@@ -0,0 +1,94 @@
+import React from "react";
+import { Droppable } from "react-beautiful-dnd";
+import BoardTask from "./Task/BoardTask";
+
+function Column(props) {
+  const { column, tasks, subTasks, handleList } = props;
+
+  const [open, setOpened] = React.useState(false);
+
+  const onButtonClick = () => {
+    setOpened(!open);
+  };
+
+  return (
+    <div className="column__container">
```

Comment:
We have been omitting container usually, but it should be fine.

Reply:
Yeah, and there's another className which is column so this is fine imo